### ISSUE TITLE
Complex structural variants like TRA or INVDUP

### ISSUE THREAD

**Issue by achilles1989** [Illumina/paragraph#36]
Hi,
how to construct a graph for complex structural variants like TRA or INVDUP from sniffles or nanosv. Could you give me  an example? And is it possible to genotype translocation between chroms now ?

Thanks!

**Comment by traxexx** [comment]
Yes. For INVDUP, there is an easier way to do this: just manually convert it to a VCF record of insertion or blockwise swap(INVDUP sequence as the insertion sequence, and change POS or END accordingly). I tested Sniffles DUP and INVDUP before, but the result is not very good. Possible reasons include:
1. They're not breakpoint accurate
2. Sniffles doesn't give the copy number. I assume they're one extra copy but that's not a good assumption

We discontinued the support for TRA/BND because without knowing what type of variants, it is dangerous to genotype only a single breakpoint. I personally would not recommend genotyping TRA unless you know exactly what they are.
